tests/e2e/pageObjects/components/insights-panel.ts

```diff
@@ -22,6 +25,12 @@ export class InsightsPanel {
         }
     }
 
+    /**
+     * get acrive tab
+     */
+    async getActiveTab(): Promise<string> {
```

Comment:
looks like better to rename it getActiveTabName()

Reply:
fixed